worlds/witness/docs/en_The Witness.md

```diff
@@ -10,12 +10,13 @@ config file.
 Puzzles are randomly generated using the popular [Sigma Rando](https://github.com/sigma144/witness-randomizer).
 They are made to be similar to the original game, but with different solutions.
 
-Ontop of that each puzzle symbol (Squares, Stars, Dots, etc.) is now an item.
+Ontop of that, each puzzle symbol (Squares, Stars, Dots, etc.) is now an item.
 Panels with puzzle symbols on them are now locked initially.
 
 ## What is a "check" in The Witness?
 
 Solving the last panel in a row of panels or an important standalone panel will count as a check, and send out an item.
+It is also possible to add Environmental Puzzles into the location pool via a setting.
```

Comment:
It may be a good idea to give the name of the setting here.

Reply:
The website display name or the yaml name?

Reply:
I'd use the website display name, as modifying a `yaml` file directly is considered "advanced" usage.